GoogleForCreators/web-stories-wp
# Docs: Add Shape Documentation and SVG Normalization Script

## Context
Documentation for future template creation

## Summary
Adds description of process for adding SVG paths as shapes for the code editor. Also included a little internal node script that allows for normalizing svg paths with a high level of precision needed to retain detail in a `0->1` coordinate space.

## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
NA


## Testing Instructions
If you got illustrator, can try out the instructions. if not you can work with the code blurbs and see that a new shape shows up in the editor.


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7228
